changelog.md

```diff
@@ -1,3 +1,52 @@
+<a name="9.6.0"></a>
+# 9.6.0 (2022-05-09)
+[Full Changelog](https://github.com/compare/v9.5.0...v9.6.0)
+
+We expect this release to ship in the DevTools of [Chrome 103](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## New Audits
+
+* add responsiveness metric for timespans ([#13917](https://github.com/GoogleChrome/lighthouse/pull/13917))
```

Comment:
You seem like you have a process going for force-pushing commits, so I'll leave this here:

The new performance metric [Interaction to Next Paint (INP)](https://web.dev/inp/) assesses responsiveness by measuring the latency of interactions throughout the page lifecycle. INP can be measured using Lighthouse timespan mode accessible through [user flows](https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md) or the [new DevTools panel](https://developer.chrome.com/blog/new-in-devtools-101/#lighthouse). ([#13917](https://github.com/GoogleChrome/lighthouse/pull/13917))

Reply:
```suggestion
* The new performance metric [Interaction to Next Paint (INP)](https://web.dev/inp/) assesses responsiveness by measuring the latency of interactions throughout the page lifecycle. INP can be measured using Lighthouse timespan mode accessible through [user flows](https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md) or the [new DevTools panel](https://developer.chrome.com/blog/new-in-devtools-101/#lighthouse). ([#13917](https://github.com/GoogleChrome/lighthouse/pull/13917))
```

Reply:
thanks!